src/main/java/co/rsk/federate/signing/hsm/client/HSMResponseCode.java

```diff
@@ -0,0 +1,51 @@
+package co.rsk.federate.signing.hsm.client;
+
+import co.rsk.federate.signing.hsm.HSMDeviceException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Kelvin Isievwore on 04/05/2023.
+ */
+public enum HSMResponseCode {
```

Comment:
Aren't we missing codes 0 and 1 here? There's a method `getOkErrorCodes` in HSMResponseHandlerBase that would make use of them. Also in PowHSMResponseHandler

Reply:
0 And 1 Are Success Codes, Not Error Codes, The Method `getOkErrorCodes` Already Handles The Success Path

Reply:
Yeah, and the enum is for HSM response codes not just error codes. I think it makes sense to have them all in one place. getOkErrorCodes can then fetch them from the enum instead of having them hardcoded